CHANGELOG.md

```diff
@@ -5,6 +5,11 @@ All notable changes to this project will be documented in this file.
 
```

Comment:
We should change this to log each version not added improved and fixed. This should be bumped for each version we have can you change that @jbernardo95 ?